step-ca/provisioners.mdx

```diff
@@ -1027,6 +1030,96 @@ To get a client certificate for a hardware-bound private key on your YubiKey:
+              "jwksUrl": "https://www.example.com/oauth2/v3/certs",
+              "userInfoUrl": "https://openidconnect.example.com/v1/userinfo",
+              "signatureAlgorithms": [
+                "RS256",
+                "ES256",
+                "ES384",
+                "EdDSA"
+              ]
+            },
+            "config": {
+                "clientId": "wireapp",
+                "signatureAlgorithms": [
+                    "RS256",
+                    "ES256",
+                    "ES384",
+                    "EdDSA"
+                ]
+            },
+            "transform": "{\"name\": \"{{ .name }}\", \"preferred_username\": \"{{ .preferred_username }}\"}"
+          },
+          "dpop": {
+            "key": "base64(public.pem)",
+            "target": "https://example.com/clients/{{ .DeviceID }}/access-token"
+          }
+      }
+    }
+}
+```
+
+* **challenges**: to enable the Wire integration, `wire-oidc-01` and `wire-dpop-01` must be set. 
+* **options**<Reference id="star6" marker="*" />: with the Wire integration enabled, the certificate template can include (transformed) properties from the OIDC and DPoP tokens (e.g. `.Oidc.preferred_username`, and `.Dpop.sub`). See [template configuration](templates.mdx#configuring-the-ca-to-use-templates) for more template configuration details.
+* **discoveryBaseUrl**<Reference id="star6" marker="*" />: sets the OIDC discovery URL, automatically configuring the OIDC provider. If provided, the other  `oidc` configuration options are ignored.
+* **issuerUrl**<Reference id="star6" marker="*" />: sets the OIDC issuer URL. Ignored if `discoveryBaseUrl` is set.
+* **authorizationUrl**<Reference id="star6" marker="*" />: sets the OIDC authorization URL. Ignored if `discoveryBaseUrl` is set.
+* **tokenUrl**<Reference id="star6" marker="*" />: sets the OIDC token URL. Ignored if `discoveryBaseUrl` is set.
+* **jwksUrl**<Reference id="star6" marker="*" />: sets the OIDC JWKS URL. Ignored if `discoveryBaseUrl` is set.
+* **userInfoUrl**<Reference id="star6" marker="*" />: sets the OIDC userInfo URL. Ignored if `discoveryBaseUrl` is set.
+* **provider.signatureAlgorithms**<Reference id="star6" marker="*" />: sets the supported OIDC signature algorithms. Ignored if `discoveryBaseUrl` is set.
+* **clientId**: sets the OIDC client ID used in OIDC token validation.
+* **config.signatureAlgorithms**<Reference id="star6" marker="*" />: sets the OIDC signature algorithms to be used in validation.
+* **transform**<Reference id="star6" marker="*" />: a Go template that can be used to map and/or transform properties in the OIDC token. Different SSO IdPs include different claims in their OIDC tokens. Using this `transform` template, the tokens are verified first, then transformed and persisted. Transformation uses the same engine as our certificate template engine. The properties from the transformed token can be referenced in the certificate template configured in the `x509` option. 
```

Comment:
Please consider adding a link to the Go template syntax.

Reply:
Added in 4c42e0dc5faaf8b42fd9c8e26e7bf7571f1a52b1.